### PR TITLE
[#29] Support GHC 9.6

### LINKED ISSUE #29: Support GHC 9.6

### PR DESCRIPTION
**Pull request by vrom911**
Resolves #29

### CODE REVIEW

**Comment by vrom911** [comment]
@alaendle please have a look!
I am going to do a release after this one, so I will write down the process andit also could be the first step for you to create releases later on as well, if you'd want to 🙂

**Review by alaendle — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,102 +3,107 @@
 `co-log-core` uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
+## 0.3.3.0 — Nov 2, 2022
```
Comment (alaendle) [inline CHANGELOG.md:6]: Need to adapt date before release is made.

**Comment by alaendle** [comment]
Btw, no hurry for a new release because thanks to @Bodigrim a revision is in place that allows ghc 9.6 (see also https://github.com/co-log/co-log-core/pull/30#issuecomment-1542846558).

File: CHANGELOG.md (hunk 1 above)
Comment (vrom911) [reply CHANGELOG.md]: Good catch!

**Comment by vrom911** [comment]
Makes sense!
Do you want to add something else in the new release?
You can add the issues in the next release, so we can work on that, or just grab the things you think we can do 🙂

**Comment by alaendle** [comment]
We could consider #34; not sure. Nothing to add to vNext from my side; I believe that `co-log-core` is in pretty good shape!

**Comment by alaendle** [comment]
Merged - but not planning a release for now - therefore I removed the version bump, cause it is unclear if the next release will contain a breaking change (0.3.3 or 0.4?). Also made a new hackage revison to allow latest `doctest`. https://hackage.haskell.org/package/co-log-core-0.3.2.0/revisions/

**Pull request merged**